scripts/core/world/world.js

```diff
@@ -136,6 +141,14 @@ define([
                     : {};
             }
 
+            this.setAgentCallback = function(cb) {
+                agentUpdateCallback = cb;
+            }
+
+            this.setTileCallback = function(cb) {
```

Comment:
Thats for when the tile will be updated but it is not used anywhere at the moment.